downgrade

```diff
@@ -271,13 +373,12 @@ cli() {
       su root -c "$cmd"
     }
   fi
-
-  main "${terms[@]}"
-  sudo pacman -U "${pacman_args[@]}" "${to_install[@]}"
-  prompt_to_ignore "${to_ignore[@]}"
 }
 
 if ((!LIB)); then
   set -e
   cli "$@"
```

Comment:
I would rename this to `parse_options` or something like that now. Or maybe even do this:

```sh
parse_options() {
  # old contents of most of cli()
}

cli() {
  parse_options "$@"
  main "${terms[@]}"
  sudo pacman -U "${pacman_args[@]}" "${to_install[@]}"
  prompt_to_ignore "${to_ignore[@]}"
}
  
if ((!LIB)); then
  set -e
  cli "$@"  # unchanged now
fi
```

Reply:
Yes this looks cleaner